src/shared/ui/components/tooltip/tooltip.component.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react';
+
+import { TooltipProperties } from './tooltip.types';
+
+export const Tooltip = ({ children, tooltipMessage }: TooltipProperties) => {
```

Comment:
for reference - it is used only to show here (Copied address!)
<img width="850" alt="image" src="https://github.com/user-attachments/assets/9ca24a5a-0e64-4905-86d2-d539addabe9f">

Reply:
totally fine, this copy does not really need a reaction at all imo

Reply:
@nemmtor ignore the comment above, just talked to @lennardevertz about this

although this feature won't be marketed, it will be more than merely an internal tool - we are going to communicate about it at least in the docs and to interested users

therefore, the UI & UX still needs to be good, but without investing too much time into it. 

1. let's either leave the tooltip as it is (totally fine), or if you insist @nemmtor let's display static text `Address copied ` in the list element + bg-green-100 applied to the background of the element for the same duration as the tooltip. We can also add closing of the dropdown after, as you suggested some time ago @nemmtor.
2. this made us realize that the copying functionality on block explorers (e.g. on Polygonscan.com) should not be there at all - the primary reason for it to exist there is to just fill the input field (as opposed to lookup widget where copying matters)

also, this comment made me do some quick visual improvements to the address lookup widget (in my PR): 6b95c8327759794ed63b7d6e0d00b4a343448e91 CC @SolutionsEngineer

Reply:
yeah so let's just display some static text instead of implementing Tooltip because proper Tooltip would take too much time

Reply:
I've removed the tooltip and added static text replacing other content as suggested.
@geoist1 do we want to address the 2nd point in this PR?

Reply:
@geoist1

Reply:
2. fixed in [294ed1d](https://github.com/idriss-crypto/browser-extensions/pull/59/commits/294ed1d6a62fb7ec5184a9305cf93530cd2627d3)